msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/OAuthAuthorizationGrant.java

```diff
@@ -15,74 +13,54 @@
 
 class OAuthAuthorizationGrant extends AbstractMsalAuthorizationGrant {
```

Comment:
q: who adds client_info = 1 ?

Reply:
It's added in the `toParameters()` method: https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/a990b4a75981d4ebd5351e978df4fb3a4c547b41/msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/OAuthAuthorizationGrant.java#L72

Looks like that line hasn't been changed since mid-2019, and other classes that extend `AbstractMsalAuthorizationGrant` do the same.